cmd/node/main.go

```diff
@@ -1481,6 +1562,35 @@ func createNodesCoordinator(
 	return nodesCoordinator, nil
 }
 
+func serializableValidatorsToValidators(nodeRegistryValidators map[string][]*sharding.SerializableValidator) (map[uint32][]sharding.Validator, error) {
```

Comment:
make the methods in the sharding folder public - no need in main

Reply:
fixed